src/charts/bar.js

```diff
@@ -210,28 +218,28 @@ define(function(require) {
         function buildContainerGroups() {
             let container = svg
                 .append('g')
-                  .classed('container-group', true)
-                  .attr('transform', `translate(${margin.left + yAxisPaddingBetweenChart}, ${margin.top})`);
+                .classed('container-group', true)
```

Comment:
Oh, what happened here?
append is a special case that gets 2 spaces instead of four.
Was this eslint?

Reply:
This happened after i ran `yarn lint:fix`

Reply:
I see, is there an easy way of rollback that change only? It messed up all the indentation.

Reply:
Rolled back before the linting 👍